examples/job/get-file-from-job.py

```diff
@@ -62,32 +60,29 @@
 from exabyte_api_client.endpoints.bank_workflows import BankWorkflowEndpoints
 from exabyte_api_client.endpoints.raw_properties import RawPropertiesEndpoints
 
+
 # ### Create and submit the job
 # 
 # For this job, we'll use the workflow located [here](https://platform.exabyte.io/analytics/workflows/84DAjE9YyTFndx6z3).
 # 
-# This workflow is a single-point total energy calculation using Density-Functional Energy as-implemented in Quantum
-# Espresso version 5.4.0.
+# This workflow is a single-point total energy calculation using Density-Functional Energy as-implemented in Quantum Espresso version 5.4.0.
 # 
 # The PBE functional is used in conjunction with an ultrasoft pseudopotential and a planewave basis set.
 # 
-# The material we will investigate is elemental [Silicon](https://materialsproject.org/materials/mp-149/), as-is from
-# Materials Project.
+# The material we will investigate is elemental [Silicon](https://materialsproject.org/materials/mp-149/), as-is from Materials Project.
 # 
-# > <span style="color: orange">Note</span>: This cell uses our API to copy the unit cell of silicon from Materials Project
-# into your account. It then copies a workflow to get the total energy of a system using Quantum Espresso to your account.
-# Finally, a job is created using the Quantum Espresso workflow for the silicon unit cell, and the job is submitted to the
-# cluster. For more information, please refer to our [run-simulation-and-extract-properties](
-# ./run-simulations-and-extract-properties.ipynb) notebook, located in this directory.
+# > <span style="color: orange">Note</span>: This cell uses our API to copy the unit cell of silicon from Materials Project into your account. It then copies a workflow to get the total energy of a system using Quantum Espresso to your account. Finally, a job is created using the Quantum Espresso workflow for the silicon unit cell, and the job is submitted to the cluster. For more information, please refer to our [run-simulation-and-extract-properties](./run-simulations-and-extract-properties.ipynb) notebook, located in this directory.
 
 # In[]:
 
 
 # Get some account information
 project_endpoints = ProjectEndpoints(*ENDPOINT_ARGS)
-project_metadata = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]
-project_id = project_metadata['_id']
-owner_id = project_metadata['owner']['_id']
+project_metadata = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[
+    0
```

Comment:
This is a weird change, should not be done

Reply:
Most of these changes are coming from our autoformatter based on linewidth of the code; should I revert the use of nb_black, or should I reformat the code to prevent it from wrapping at the list index being specified?